### DISCUSSION TITLE
Product Filter by WBW issues

### DISCUSSION THREAD

**Discussion by goclimbing** [bootscore/bootscore#902] [Q&A]
In this thread near the bottom Basti and I get into looking at the problems I'm experiencing with the Product Filter plugin.   I'm moving that discussion to here.

https://github.com/orgs/bootscore/discussions/896#discussioncomment-11579356

This is the site before switching to bootscores. It's running understrap.  The plugin is widely used and gets great reviews. So something in the theme.

https://stg-psbullion-dev.kinsta.cloud/shop/

Site where it doesn't work:
https://psbullion.kinsta.cloud/shop/

select any check box on the latter site and it reloads the page rather than just reloading the content.

So far I've tried.

>- Disable AJAX cart with this two filters as described here https://bootscore.me/documentation/woocommerce/#restoring-woocommerce-s-default-functionality

Disabling AJAX had no effect.

>- Comment each line in functions.php Lines 37 to 61 Lines 18 to 29 and wc-functions.php until you have found it:

Commenting out those lines did not change the behavior.  There were couple lines I could not comment out as it would throw a fatal error.  Those are lines:
47 require_once('inc/navwalker.php');               // Register the Bootstrap 5 navwalker
53 require_once('inc/widgets.php');                 // Register widget area and disables Gutenberg in widgets

25 require_once('inc/wc-setup.php');

I have also reset the child theme to the original eliminating any of my code.

On my local dev env i have disabled all plugins other than the filter and WC.

We have the pro version so I might check with their support to see if they could take a look. Obviously no promises there.

**Comment by crftwrk** [comment]
It seems that this has something to do in `woocommerce/loop/` files, but I can't figure out this quickly. 

Installed the plugin here https://dev.bootscore.me/shop/ with shortcode in the sidebar and works. There are settings in the plugin options tab where you can disable AJAX or define a container selector:


| Disable AJAX  | Define container |
| ------------- | ------------- |
| <img width="415" alt="Bildschirmfoto 2024-12-17 um 10 56 13" src="https://github.com/user-attachments/assets/58278152-7fd3-40e1-ba48-ecc1c4030f32" />  | <img width="795" alt="Bildschirmfoto 2024-12-17 um 11 05 09" src="https://github.com/user-attachments/assets/d1e68b1f-288e-4b53-bd61-07bb27346895" />  |

It seems that the default product class in `loop-start.php` or `content-product.php` is missing. This settings are a workaround, but not a solid solution. It would be great if you can compare this files from Understrap/Bootscore and check what is the difference. A PR is highly appreciated.

**Reply by crftwrk** [reply]
Got it. Plugin uses `ul.procucts` for default container, but Bootscore uses a `div` here. So, we have to change the `div` to `ul` here:

https://github.com/bootscore/bootscore/blob/901372fe9d1390531f468941eafeaf991f06b53a/woocommerce/loop/loop-start.php#L24

https://github.com/bootscore/bootscore/blob/901372fe9d1390531f468941eafeaf991f06b53a/woocommerce/loop/loop-end.php#L23

And change `div` to `li` here:

https://github.com/bootscore/bootscore/blob/901372fe9d1390531f468941eafeaf991f06b53a/woocommerce/content-product.php#L29

https://github.com/bootscore/bootscore/blob/901372fe9d1390531f468941eafeaf991f06b53a/woocommerce/content-product.php#L75

https://github.com/bootscore/bootscore/blob/901372fe9d1390531f468941eafeaf991f06b53a/woocommerce/content-product-cat.php#L23

https://github.com/bootscore/bootscore/blob/901372fe9d1390531f468941eafeaf991f06b53a/woocommerce/content-product-cat.php#L65

Want to contribute and create a PR?
